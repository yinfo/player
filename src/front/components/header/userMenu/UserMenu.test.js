// @flow

import React from 'react';
import { shallow } from 'enzyme';
import UserMenu from './UserMenu';

describe('UserMenu component', () => {
  const props = {
    login: 'login',
    picture: 'some_picture_link',
    firstname: 'firstname',
    lastname: 'lastname',
    onLogout: jest.fn(),
  };

  it('renders as expected', () => {
    const component = shallow(<UserMenu {...props} />);
    expect(component).toMatchSnapshot();
  });
});
