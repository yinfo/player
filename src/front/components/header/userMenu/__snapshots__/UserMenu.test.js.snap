// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserMenu component renders as expected 1`] = `ShallowWrapper {}`;
